game_event/views.py

```diff
@@ -1 +1,37 @@
-# from django.shortcuts import render
+from django.shortcuts import render
+from .models import GameEvent
+from datetime import date
+
+
+def game_events(request):
+    game_events = GameEvent.objects.all()
+
+    if request.GET.get('min_players'):
+        try:
+            min_players = int(request.GET.get('min_players'))
+            if min_players > 0:
+                game_events = game_events.filter(min_number_of_players__gte=min_players)
+        except ValueError:
```

Comment:
which line is expected to generate this exception? You should have only that line within the try block. Same goes for the filters below.